src/main/scala/Getsetapi/Example.scala

```diff
@@ -0,0 +1,139 @@
+package Getsetapi
+
+import scala.collection._
+import scala.collection.JavaConverters._
+import org.eclipse.jetty.server.{Connector, NetworkConnector, Server, ServerConnector}
+import javax.servlet.http.{HttpServlet, HttpServletRequest, HttpServletResponse}
+import org.eclipse.jetty.servlet.ServletHandler
+import java.util.concurrent.ConcurrentHashMap
+
+import org.eclipse.jetty.server.nio.NetworkTrafficSelectChannelConnector
+
+
+object Example {
+
+  import Mymap._
+  import org.eclipse.jetty.util.thread.QueuedThreadPool
+
+  val threadPool = new QueuedThreadPool(6, 1)
+  val server = new Server(threadPool)
+  var connector = new ServerConnector(server)
+  connector.setPort(8090)
+  server.setConnectors(Array(connector))
+  val getRoute = "/get"
+  val putRoute = "/put"
+  val updateRoute = "/update"
+  val deleteRoute = "/delete"
+  val handler = new ServletHandler()
+
+  def foo(x: Array[String]) = x.foldLeft("")((a, b) => a + b)
+
+  def main(args: Array[String]) {
+    println("Hello World!")
+    println("concat arguments = " + foo(args))
+
+    server.setHandler(handler)
+    handler.addServletWithMapping(classOf[getServlet], getRoute)
+    handler.addServletWithMapping(classOf[putServlet], putRoute)
+    handler.addServletWithMapping(classOf[updateServlet], updateRoute)
+    handler.addServletWithMapping(classOf[deleteServlet], deleteRoute)
+
+    server.start()
+    println(s"Server started on ${port()} with Routes: '$getRoute', '$putRoute', '$updateRoute', '$deleteRoute'")
+    server.join()
+
+  }
+
+
+  def port() = 8090
+
+  object Mymap {
```

Comment:
Classes and Objects should be PascalCase.

Reply:
Could this be a separate class in another file and the object here instantiates it? Instead of nesting classes and object, it's cleaner to define them independently and then instantiate wherever needed. This increases code reusability as well.